static/js/manage/backlog-overview.js

```diff
@@ -1,4 +1,60 @@
 MEDUSA.manage.backlogOverview = function() {
+
+    $(document).ready(function() {
```

Comment:
If it's not working without then it needs to be fixed before merging as the whole `MEDUSA` object only runs when the page is loaded.